### PR TITLE
refactor: Update fetching instructions in test PR guide

### LINKED ISSUE #53: Update test PR guide following most recent mentorship-backend docs change
## Description

Update this repository instructions to fetch a contributor's PR: https://github.com/anitab-org/documentation/blob/master/quality-assurance.md#how-to-test-a-pr
You can follow this PR work https://github.com/anitab-org/mentorship-backend/pull/1076

## Mocks

https://github.com/anitab-org/mentorship-backend/pull/1076

### Update [Required]
- [ ] Update test PR guide using info provided in the description

## Definition of Done
- [ ] All of the required items are completed.
- [x] Approval by 1 mentor.

## Estimation
0.5 hours

### PR DESCRIPTION
**Pull request by StephNathai**
### Description
Updated the fetching instructions to test a PR in the [quality-assurance.md](https://github.com/anitab-org/documentation/blob/master/quality-assurance.md#how-to-test-a-pr) file

Fixes #53 

### Type of Change:
- Documentation

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)

### How Has This Been Tested?
See file and display rendered version of MD file.

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

**Code/Quality Assurance Only**

- [x] My changes generate no new warnings

### CODE REVIEW

File: quality-assurance.md
Comment (isabelcosta) [inline quality-assurance.md:51]:
```suggestion
  git fetch upstream pull/<id>/head:<branch_name>
```
just because above you mention `<id>`

File: quality-assurance.md
Comment (isabelcosta) [inline quality-assurance.md:54]:
```suggestion
5. Alternatively, if you use the [GitHub CLI](https://cli.github.com/), you can run the following command:
```

**Review by isabelcosta — CHANGES_REQUESTED** [review]
This is great @StephNathai ! 🙌🏾 
I only left a couple of comments to make the instructions more clear and I can approve after.

Thank you for contributing!

**Review by isabelcosta — APPROVED** [review]
I saw myself needing this guide, so I've updated the changes I requested 😅 
Thank you for contributing @StephNathai 🙌🏾

**Comment by StephNathai** [comment]
No worries! I made the changes but was waiting on an internal review before I could push it externally. Thanks!

**Comment by isabelcosta** [comment]
@StephNathai what do you mean by internal review? You can push right after you do the changes :)

**Comment by StephNathai** [comment]
@isabelcosta I'm contributing on behalf of my company so my company needs to approve all changes internally before we push them up externally

**Comment by isabelcosta** [comment]
Ah ok! I just merged it 💪🏾 Thank you for your contribution! @StephNathai

**Pull request merged**
